snippets/c++/Longest_common_subsequence/lcs.md

```diff
@@ -0,0 +1,82 @@
+# Longest Common Subsequence
```

Comment:
Now make the changes suggested by Saksham and your PR will be merged :-)

Reply:
@kaustubhgupta @sakshamarora1 
I made the suggested changes. Please review :)